lib/runner.go

```diff
@@ -18,18 +18,16 @@ import (
 )
 
 const (
-	unknownExitCode = -1
+	unknownExitCode  = -1
+	skipTestExitCode = 99
 )
 
 var (
-	// Green is a convenient color helper.
-	Green = color.New(color.FgGreen).SprintfFunc()
-	// GreenBold is a convenient color helper.
-	GreenBold = color.New(color.FgGreen, color.Bold).SprintfFunc()
-	// Red is a convenient color helper.
-	Red = color.New(color.FgRed).SprintfFunc()
-	// RedBold is a convenient color helper.
-	RedBold = color.New(color.FgRed, color.Bold).SprintfFunc()
+	green      = color.New(color.FgGreen).SprintfFunc()
+	greenBold  = color.New(color.FgGreen, color.Bold).SprintfFunc()
+	yellowBold = color.New(color.FgYellow, color.Bold).SprintfFunc()
+	red        = color.New(color.FgRed).SprintfFunc()
+	redBold    = color.New(color.FgRed, color.Bold).SprintfFunc()
```

Comment:
Changing the capitalization from `Green` to `green` changes the variable from public to internal.
Is that intended ?

Reply:
Yes, they should be private to the package.

Reply:
Ok